# Validation exception
class ValidationError(Exception):
  """Raised exception for non-valid check."""
  pass